app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/MoreBonusViewHeader.kt

```diff
@@ -0,0 +1,339 @@
+package com.aptoide.android.aptoidegames.drawables.backgrounds
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.PathFillType.Companion.EvenOdd
+import androidx.compose.ui.graphics.PathFillType.Companion.NonZero
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.StrokeCap.Companion.Butt
+import androidx.compose.ui.graphics.StrokeJoin.Companion.Miter
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.group
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.aptoide.android.aptoidegames.theme.Palette
+
+@Preview
+@Composable
+private fun MoreBonusViewHeaderPreview() {
+  Image(
+    imageVector = getMoreBonusViewHeader(Color.Green, Color.Magenta, Color.Gray),
+    contentDescription = null,
+    modifier = Modifier.size(240.dp)
+  )
+}
+
+fun getMoreBonusViewHeader(
+  backgroundColor: Color = Palette.Secondary,
```

Comment:
Let's follow the convention we have about colors by naming them as `color1`, `color2` etc.
Reference to `getPaymentsProgressBackground` as an example.

Reply:
Fixed